src/explorer/nodeBase.ts

```diff
@@ -23,4 +23,7 @@ export class NodeBase {
     async getChildren(azureAccount: AzureAccountWrapper): Promise<NodeBase[]> {
         return [];
     }
+
+    openInPortal?(azureAccount: AzureAccountWrapper): void {

```

Comment:
I don't think the azureAccount is ever used in these functions, so you could remove it from the signature